zerver/lib/webhooks/git.py

```diff
@@ -163,6 +190,22 @@ def get_commits_content(commits_data, is_truncated=False):
         ).replace('  ', ' ')
     return commits_content.rstrip()
 
+
 def get_short_sha(sha):
     # type: (Text) -> Text
     return sha[:7]
+
+
+def get_all_committers(commits_data):
+    # type: (List[Dict[str, Any]]) -> Dict[str, int]
+    committers = defaultdict(int)  # type: Dict[str, int]
+    for commit in commits_data:
+        committers[commit['name']] += 1
+    committers_names = list(sorted(committers.keys()))  # type: List[str]
```

Comment:
i added one test case for bitbucket2 just to check if it worked correctly.
I'll add them for the rest ;)